core/ir/arm/instr.c

```diff
@@ -32,28 +32,25 @@
 
 #include "../globals.h"
 #include "instr.h"
+#include "encode_api.h"
 #include "decode.h"
 
 /* FIXME i#1551: add A64 and Thumb support throughout */
```

Comment:
This comment can be removed: it is stale.

Reply:
Done.